README.md

```diff
@@ -7,182 +7,119 @@
 [![Coverage Status](https://coveralls.io/repos/github/linthtml/linthtml/badge.svg?branch=develop)](https://coveralls.io/github/linthtml/linthtml?branch=develop)
 <!-- [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Flinthtml%2Flinthtml.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Flinthtml%2Flinthtml?ref=badge_shield) -->
 
-> An unofficial html5 linter and validator.
+> An unofficial HTML5 linter and validator.
 
-LintHTML is a fork of [htmllint](https://github.com/htmllint/htmllint) with an integrated CLI, all htmllint's rules can be used with LintHTML.
-The migration from htmllint to LintHTML is easy, just follow this [guide](htmllint-migration).
-<!-- _LintHTML uses [htmlparser2](https://www.npmjs.com/package/htmlparser2) to parse your html._ -->
+LintHTML is a fork of [htmllint](https://github.com/htmllint/htmllint) featuring a built-in CLI and multiple bug fixes and improvements.
+The migration from htmllint to LintHTML is easy, as all htmllint's rules can be used with LintHTML – just follow this [simple guide](#migrate-from-htmllint).
 
 ## Installation and Usage
 
-Prerequisites:
```

Comment:
We should keep the prerequisites and clearly indicate the node versions supported

Reply:
Do we? Most projects don't state it in the `README` as it is not vital information (as long as it works with the latests Node.js versions, of course). In case you need more specific info, people can look in the `package.json` (engines -> node). Actually, the `README` info isn't accurate: it states that works with Node 6 onwards (6+) and actually doesn't run in Node 6. If you still prefer to keep it, we should update it to 8+, shouldn't we?

Reply:
The current `README` is highly inspired by the ESlint one which have a prerequisites part. But I don't really mind removing it.

Reply:
In the case of ESLint it makes more sense due to a specific minor version requirement. They say that it requires  `^8.10.0`, which means that 8.0.0 doesn't work and that they are aware of it. If that was the case for LintHTML I'd support noting that information, but as far as I know we aren't aware of any 8+ version incompatibilities, so I'd just drop it.